widgets/filter/src/Filter.tsx

```diff
@@ -175,6 +175,7 @@ export default class Filter extends React.Component {
                                     .concat(fetchIncludeExtra || [])
                                     .join()
```

Comment:
I'm wondering if there should not be 

```suggestion
                                    .join(',')
```

to join using a comma :smile: I did not check, but it looks suspicious to me

Reply:
Nevermind, `,` is a default separator in lodash's `join`: https://lodash.com/docs/4.17.15#join

Optional:

I would still pass it explicitly, as it looks confusing to me :smile:

Reply:
No, i don't change any in this line, for clean-history let me don't adding anything in this line

Reply:
Ok :slightly_smiling_face: